### PR TITLE
Letter 'a' from #428 (replaces #430)

### PR DESCRIPTION
**Pull request by luzpaz**
(FYI: This PR is a revision on #430 whereby non-'a' letters have been removed and other tweeks have been made)
Started to separate #428 in to discrete PRs for each letter. Also removed/corrected some entries that were not relevant/precise. This PR has been sorted as well.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -467,35 +565,52 @@ alocating->allocating
...
 alogrithm->algorithm
 alogrithms->algorithms
 alomst->almost
+aloows->allows
 alot->a lot, allot,
 alotted->allotted
 alow->allow
 alowed->allowed
 alowing->allowing
+alowinserts->allowinserts
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: To be removed

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -633,30 +783,52 @@ appeareance->appearance
...
 apperture->aperture
+appied->applied
+applay->apply
+appliable->applicable
+applicabel->applicable
 applicaiton->application
 applicaitons->applications
+applicaten->application
+applicatin->application
+applicationevnet->applicationevent
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: to be removed

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: removed

**Comment by luzpaz** [comment]
@EdwardBetts do you mind reviewing this PR? Thanks!

**Comment by luzpaz** [comment]
@larsoner would you like me to further break this PR apart so it's not so large ?

**Comment by larsoner** [comment]
Now it needs a rebase at least

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -646,31 +795,52 @@ appeareance->appearance
...
+applicabel->applicable
 applicaiton->application
 applicaitons->applications
+applicaten->application
+applicatin->application
 appliction->application
 applictions->applications
+applikation->application
 appling->applying, appalling,
+applyable->applycable
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: This should be applyable->applicable.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -692,14 +862,18 @@ approxiamtion->approximation
...
 aprehensive->apprehensive
+aproach->approach
 apropriate->appropriate
 apropriately->appropriately
 aproval->approval
 aproximate->approximate
 aproximately->approximately
 aproximation->approximation
 aproximations->approximations
+aprrovement->approvement
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: The word approvement is archaic, it isn't known by my the Firefox spell checker. I feel like we shouldn't be encouraging programmers to use it. Approval is a better choice of word.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -597,20 +729,28 @@ announcments->announcements
...
 ansalization->nasalization
 ansestors->ancestors
+ansynchronous->asynchronous
+antaliasing->antialiasing
 antartic->antarctic
+anthor->another
 anthromorphization->anthropomorphization
 anthropolgist->anthropologist
 anthropolgy->anthropology
+antialialised->antialialiased
```
Comment (EdwardBetts) [inline codespell_lib/data/dictionary.txt]: Antialialiased isn't a word, it should be antialiased.

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Good point. Changed

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Good catch. Fixed

**Pull request merged**